model_api/python/openvino/model_api/models/types.py

```diff
@@ -172,7 +172,9 @@ def __init__(
     def from_str(self, value):
         try:
             floats = [float(i) for i in value.split()]
-            ints = [int(i) for i in value.split()]
+            # In certain instances when a string contains a list of floats like "1.0 2.0 3.0",
+            # directly converting to ints will raise a ValueError
+            ints = [int(float(i)) for i in value.split()]
```

Comment:
Here is even better idea. You can wrap `ints = [int(i) for i in value.split()]` with `try except ValueError`. With that you won't need to explain that the code may throw ValueError because, well, you explicitly state that with the implementation. And in comments you may still explain why you expectb `ValueError` to be risen.